Library/Homebrew/requirements/perl_requirement.rb

```diff
@@ -0,0 +1,28 @@
+class PerlRequirement < Requirement
+  fatal true
+  default_formula "perl"
+
+  def initialize(tags)
+    @version = tags.shift if /(\d.\d{1,2})/ === tags.first
```

Comment:
- You need to escape the dot in the regular expression: `.` ➡️ `\.`.
- Aside from that, is there a reason not to make that `/^\d+\.\d+$/`?
  - We don't really care how many digits there are in the individual parts of the version number, but not making this assumption makes things more robust.
  - The surrounding parentheses are also not needed.
  - We probably don't want to match things like `garbage5.18more-garbage`, hence the anchors.